plugins/extractors/bigquery/bigquery.go

```diff
@@ -219,17 +223,27 @@ func (e *Extractor) Extract(ctx context.Context, emit plugins.Emit) error {
 			return err
 		}
 
+		ch := make(chan *bigquery.Dataset, len(datasets))
+
+		numWorkers := 5
```

Comment:
Could you please explain why we hardcoded it to 5? I don't see the concurrency variable being used anywhere